cmd/controller/controller.go

```diff
@@ -197,7 +197,7 @@ func (c *Controller) processNextItem() bool {
 	return true
 }
 
-func (c *Controller) unseal(key string) error {
+func (c *Controller) unseal(key string) (err error) {
```

Comment:
Is the named return used? I can't see a naked `return` which makes this change look a bit accidental. Or ... perhaps the new deferred bit is supposed to leave this set on the way out?

Reply:
the deferred bit reads the value of the return value and passes it down to `c.updateSealedSecretStatus(ssecret, err)`

I changed the code so that it's obvious what is happening without having to deeply understand how Go variable scopes work.

Reply:
Thanks! It seems the subtlety got lost in the a sea of `err` before, renaming it helps it stand out.